package.json

```diff
@@ -4,6 +4,15 @@
   "description": "Seamless REST/GraphQL API mocking library for browser and Node.js.",
   "main": "./lib/index.js",
   "types": "./lib/index.d.ts",
+  "exports": {
+    ".": "./lib/index.js",
+    "./iife": "./lib/iife/index.js",
+    "./native": "./lib/native/index.js",
+    "./node": {
+      "import": "./lib/node/index.mjs",
+      "require": "./lib/node/index.js"
```

Comment:
is it worth moving these to 

`require` and `default` instead of explicitly using import?

Reply:
not really should be fine either way.  I tend to think of it as ideally everything should go to esm, and `require` is the special case, so it kind of bakes that thinking into the configuration - but that's only semantic and not functional